app/controllers/restaurants_controller.rb

```diff
@@ -47,11 +47,14 @@ def check_for_exisiting_restaurant
     end
   end
 
+  def restaurant_categories
+    @categories = ['Pizza', 'Sushi', 'Italian', 'Thai', 'Fast Food', 'BBQ', 'French', 'Traditional', 'Vegan', 'Seafood', 'Texmex']
```

Comment:
My point exactly. I would call the constant PERMITTED _CATEGORIES and add a validation